policies/policies.go

```diff
@@ -26,18 +26,17 @@
 package policies
 
 import (
-	"context"
-	"fmt"
-	"os"
-	"strings"
-
 	"clouditor.io/clouditor/api/assessment"
 	"clouditor.io/clouditor/api/evidence"
-	"github.com/fatih/camelcase"
+	"clouditor.io/clouditor/internal/util"
```

Comment:
can you do a quick go fmt on this file? seems that imports are ordered wrong here.

Reply:
I've done 'go fmt'. I don't know why the order is not correct.